packages/mui-codemod/src/v6.0.0/all/deprecations-all.js

```diff
@@ -0,0 +1,11 @@
+import transformerListItemButtonProps from '../list-item-button-prop/list-item-button-prop';
+
+/**
+ * @param {import('jscodeshift').FileInfo} file
+ * @param {import('jscodeshift').API} api
+ */
+export default function deprecationsAll(file, api, options) {
+  file.source = transformerListItemButtonProps(file, api, options);
```

Comment:
This should be added to the `v6-all` one, as it's not a deprecation. And we should remove this file.